app/lib/project/MetapolatorProject.js

```diff
@@ -594,24 +598,43 @@ define([
           ;
 
         // create a bare ufoV2 directory
-        this._io.mkDir(false, dirName);
+        io.mkDir(false, dirName);
 
         // create dirName/metainfo.plist
-        this._io.writeFile(false, dirName+'/metainfo.plist'
+        io.writeFile(false, dirName+'/metainfo.plist'
                                 , plistLib.createPlistString(metainfoV2));
 
         // fontforge requires a fontinfo.plist that defines unitsPerEm
-        this._io.writeFile(false, dirName+'/fontinfo.plist'
+        io.writeFile(false, dirName+'/fontinfo.plist'
                                 , plistLib.createPlistString(minimalFontinfo));
 
-        this._io.mkDir(false, dirName+'/glyphs');
-        this._io.writeFile(false, dirName+'/glyphs/contents.plist', plistLib.createPlistString({}));
+        io.mkDir(false, dirName+'/glyphs');
+        io.writeFile(false, dirName+'/glyphs/contents.plist', plistLib.createPlistString({}));
 
-        glyphSet = this.getNewGlyphSet(
-                                false, dirName +'/glyphs', undefined, 2);
+        glyphSet = GlyphSet.factory(false, io, dirName+'/glyphs', undefined, 2);
 
         exportController = new ExportController(master, model, glyphSet, precision);
         exportController.export();
+    }
+
+    _p.exportInstance = function(io, masterName, instanceName, precision, dataType){
+        if (instanceName.slice(-4) === '.zip'){
+            var zipped = this.getZippedInstance(masterName, instanceName, precision, dataType);
```

Comment:
`var zipped = this.getZippedInstance(masterName, instanceName, precision, 'nodebuffer');`